### DISCUSSION TITLE
A Web Analytics issue, wrong statistics

### DISCUSSION THREAD

**Discussion by Crushdada** [vercel/community#6226] [Bug Report]
### Page to Investigate

https://vercel.com/davidchens-projects-68c87733/binary-tree-visualization/analytics

### Steps to Reproduce

I don't know if I can reliably reproduce this problem, but today I looked at my website data as usual and found this problem. The number of visits was in the single digits, but the growth was in the thousands.
Post a screenshot below 👇

![img_v3_0298_07707746-060d-4f55-b7c8-810079cadbeg](https://github.com/vercel/community/assets/73060999/bc2bed7b-4de7-4b74-b692-d8b9c3b4a6f0)

**Comment by amyegan** [comment]
Thanks for reporting this. I believe it was related to an incident which has since been resolved: https://www.vercel-status.com/incidents/q93stnxj3rkg 

Please reach out again if you have any further problems with this 🙏
